learn/installer/install-and-run.md

```diff
@@ -0,0 +1,48 @@
+# Quick Start
+
+Codex Installer is a CLI tool that simplifies the process of setting up a Codex node. It handles all the necessary steps to get your node running quickly and efficiently.
+
+> Before we get started, please make sure to reivew and accept the [disclaimer](/codex/installer-disclaimer) as Codex Installer collects some of your node information to improve the user experience.
+
+<br/>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/CcFtQzmzGSg?si=wUHfIgGWggIcyzqT" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## Requirements
+
+- git
+- npm
+
+## Run the Codex Installer
+
+To start the Codex Installer, simply run the following command in your terminal :
+
+```bash
+npx codexstorage
+```
+
+## Download and install Codex
+
+Once you run the above command, you will be seeing a command line interface with various options as below :
+
+![InstallCodex](/learn/codex-installer.png)
+
+Select the first option to download and install Codex. You will be asked to agree to the privacy disclaimer and provide an installation path to finish your installation.
+
+## Run the Codex node
+
+Upon selecting the second option, you will be asked for your Listening port (default is 8070) and discovery port (default is 8090). You can optionally provide your ERC20 public address to associate your testnet node with your wallet (Please note that Codex does not promise any incentives for running a node yet).
```

Comment:
I would definitely avoid giving the explicit number of the option in the menu. If those get moved, the docs are outdated and possibly confusing.

Reply:
Agreed. I will be mindful about this in the future and fix this atm